Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.py

```diff
@@ -84,7 +84,7 @@ def get_azure_download_link(self):
             timeout=self._polling_timeout,
             resp_type='text',
             retries=4,
-            status_list_to_retry=[403]
+            status_list_to_retry=[403, 404]
```

Comment:
To avoid the 404 error
https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/13317625#L1575